robottelo/ui/packages.py

```diff
@@ -51,3 +51,13 @@ def check_file_list(self, name, file_list):
         for package_file in file_list:
             self.wait_until_element(
                 locators['package.content_file'] % package_file)
+
+    def search_in_repo(self, name, repo_name, expecting_results=True):
```

Comment:
In that case we should use simple search. Actually, it is possible to pass `All repositories` as `repo_name` but it just has no sense :)